.travis.yml

```diff
@@ -1,7 +1,15 @@
 sudo: true
 language: ruby
-rvm:
-  - 2.3.1
 before_install:
-  - gem install bundler -v 1.13.6
+  - "find /home/travis/.rvm/rubies -wholename '*default/bundler-*.gemspec' -delete"
```

Comment:
Does `gem uninstall` not work?

Reply:
Unfortunately not.  I tried in the `before_install` and `install` phases. Travis returns:

```
$ gem uninstall bundler
ERROR:  While executing gem ... (Gem::InstallError)
    gem "bundler" cannot be uninstalled because it is a default gem
```

[Related issue](https://github.com/travis-ci/travis-ci/issues/8717)